android/app/src/gps/java/covidsafepaths/gps/bridge/SecureStorageManager.kt

```diff
@@ -27,7 +27,6 @@ class SecureStorageManager(reactContext: ReactApplicationContext) : ReactContext
 
   @ReactMethod
   fun removeAllLocations(promise: Promise) {
-//    TODO: Implement the following method in SecureStorage
-//    SecureStorage.removeAllLocations(promise)
+    SecureStorage.removeAllLocations(promise)
```

Comment:
Great! To confirm, this should automatically work with no RN changes needed? @tnekret

Reply:
Yes, and as soon as this is merged i'll test locally before merging my PR with RN code